src/_data/side-nav.yml

```diff
@@ -49,15 +49,17 @@
   - title: "Dart Testing"
     permalink: /guides/testing
 
-- title: "Resources"
+- title: Resources
   children:
-  - title: "Codelabs"
+  - title: Install
+    permalink: /install
+  - title: Codelabs
     permalink: /codelabs
-  - title: "Tutorials"
+  - title: Tutorials
     permalink: /tutorials
-  - title: "Articles"
+  - title: Articles
     permalink: /articles
-  - title: "Dart Tools"
+  - title: Tools
```

Comment:
Similarly, I expected Tools to be bold when I was on a page under it. I don't know if it's feasible to add all those pub pages to the sidenav.

Reply:
I'll fix this through https://github.com/dart-lang/site-www/issues/403